core/tx_pool.go

```diff
@@ -162,6 +171,8 @@ var DefaultTxPoolConfig = TxPoolConfig{
 	GlobalQueue:  1024,
 
 	Lifetime: 3 * time.Hour,
+
+	Blacklist: &map[common.Address]bool{},
```

Comment:
Use struct{} not bool

Reply:
will change, didn't realize empty structs use no mem.